### PR TITLE
Verified commits

### PR DESCRIPTION
**Pull request by sebastinez**
This PR adds a verified badge to all the commits that have additional peer information, which means that the `http-api` was able to match the commit signature fingerprint to a key available on the seed.

**Things to fix:**
- [x] Fix test suite
- [x] Give some types better names 🤷‍♂️

### CODE REVIEW

**Comment by cloudhead** [comment]
Looks good so far.

**Comment by sebastinez** [comment]
Okay test suite works now and I think names make sense.

**Comment by sebastinez** [comment]
Also moves the `navigateHistory` helper function from the `History` page to the `CommitTeaser`.
Allows us to remove one of div wrappers and just have one `div.commit` instead of two.

File: src/commit.ts [hunk 1]
```diff
@@ -27,6 +32,20 @@ export interface GroupedCommitsHistory {
   stats: Stats;
 }
 
+export interface Person {
+  name: string;
+}
+
+export interface CommitContext {
+  committer: {
```
Comment (cloudhead) [inline src/commit.ts:40]: Should this be optional?

**Review by cloudhead — APPROVED** [review]

**Pull request merged**